### PR TITLE
[shelly] improved logging on WebSocket connection problems

### PR DESCRIPTION
**Pull request by markus7017**
This includes
-  improved logging on WebSocket connection problems
- removing a duplicate resource on shelly.properties
- some minor (cosmetic) code improvements

### CODE REVIEW

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java [hunk 1]
```diff
@@ -700,10 +700,14 @@ public void onMessage(String message) {
...
     @Override
-    public void onClose(int statusCode, String reason) {
+    public void onClose(int statusCode, String description) {
         try {
-            logger.debug("{}: WebSocket connection closed, status = {}/{}", thingName, statusCode, getString(reason));
-            if (statusCode == StatusCode.ABNORMAL && !discovery && getProfile().alwaysOn) { // e.g. device rebooted
+            String reason = getString(description);
+            logger.debug("{}: WebSocket connection closed, status = {}/{}", thingName, statusCode, reason);
+            if ("Bye".equalsIgnoreCase(reason)) {
+                logger.debug("{}: Device went to sleep mode", thingName);
```
Comment (lsiepel) [inline bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java:708]: Just a check, if the device is in sleep mode, it is still responsive? As in you don't need to set the Thing status?

Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java:708]: No, when it sleeps it needs an event to wake-up  (sensors are still active, but the device is not accessible via the APII), e.g. the temp sensor reports a change then the device wakes up and sends an event message to the binding.

Maybe I should change the message to "Connection terminated, e.g. device in sleep mode"?

File: bundles/org.openhab.binding.shelly/src/main/resources/scripts/oh-blu-scanner.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  * This script uses the BLE scan functionality in scripting to pass scan reults to openHAB
```
[the remark this answers is not in the document]
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/resources/scripts/oh-blu-scanner.js]: @lsiepel fixed

File: bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java (hunk 1 above)
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiRpc.java:708]: @lsiepel changed

**Review by lsiepel — APPROVED** [review]
LGTM

**Pull request merged**